features/learner_changes_employer.feature

```diff
@@ -101,3 +101,43 @@ Feature: Provider earnings and payments where a learner changes employers
             | SFA Levy employer budget   | 1000  | 1000  | 0     | 0     | 0     |
             | SFA Levy co-funding budget | 0     | 0     | 0     | 0     | 0     |
 
+
+
+
+			
+    Scenario: Earnings and payments for a DAS learner, levy available, where a learner switches from DAS to Non Das employer at the end of month
+        Given The learner is programme only DAS 
+        And the ABC has a levy balance > agreed price for all months
+    		
+		And the learner changes employers
```

Comment:
in Visual studio and Notepad++ it looks ok, so not really sure what to fix here.